hushline/routes.py

```diff
@@ -516,10 +524,23 @@ def logout() -> Response:
     @app.route("/directory")
     def directory() -> Response | str:
         logged_in = "user_id" in session
+
+        intro_text = (
+            intro_text_setting.value
+            if (
+                intro_text_setting := OrganizationSetting.fetch_one(
+                    OrganizationSetting.DIRECTORY_INTRO
+                )
+            )
+            and intro_text_setting.value
+            else None
+        )
+
         return render_template(
             "directory.html",
             usernames=get_directory_usernames(),
             logged_in=logged_in,
+            intro_text=intro_text,
```

Comment:
Intro text must be escaped here also. User input into the DB should be (almost) always be treated as untrusted. If we want to pre-clean/sanitize it going in, that's fine, but at display time we always need to do it again. Also wrap it in `Markup`

Reply:
323efb6